Recalculated Teams Table Fielding Percentage Precision

orrski opened the pull request:
I recalculated the fielding percentages for Teams table for all years before 2000 (through 1999).

tturocy commented:
Could you confirm the source for these - is it the old databank data?  Also, it seems if we are going to standardise on three decimal places, we should do it in every row (some of them are still reported with two or even one digit, which I assume are cases of trailing zeroes).

orrski commented:
My source: I used the Fielding table. I did not pull the numbers from a previous version of the database.

However, I was originally double-counting many of the outfield results (as I did in closed pull request #16). I addressed that particular double-counting in a subsequent commit, which is now included in this pull request.

I also added in the missing trailing zeros. I believe all the fielding percentages are now displayed to three decimal places.

Are there any other "gotchas" in the Fielding table that I missed?

tturocy commented:
I don't think one can compute the team fielding percentages by using the Fielding table.  Although in theory statistics should balance between team and players, in practice they don't. In addition, this is a dataset that does not have the cleanest provenance; we really don't know if or how many fielding records may have been dropped or corrupted in the various iterations over the years.

I think the only proper thing to do would be to restore the team fielding percentages from the last version of the data before these were truncated, whenever that happened.
